R/carehomes.R

```diff
@@ -1356,9 +1362,10 @@ carehomes_parameters_progression <- function(dt,
               gamma_ICU_pre = 2,
               gamma_W_R = 2 / 5,
               gamma_W_D = 2 / 5,
-              gamma_sero_pre_1 = 1 / 5,
+              gamma_sero_pre_1 = 1 / 10,
```

Comment:
maybe I missed something but why were there already 2 gamma_sero_pre_1 and gamma_sero_pre_2? 
And why has the value of gamma_sero_pre_1 changed but not gamma_sero_pre_2?
If I understand correctly, you have split each of sero_pre_1 and sero_pre_2 in 2 successive compartments since the k parameters for those are now 2, so should the duration of each be half, and therefore the new gamma be twice? In that case I would expect where you had 1 / 5 you now should have 1 / 2.5? but maybe I'm misunderstanding something

Reply:
These are defaults, which we change within the transform. But will be worth updating some of the defaults to avoid need to change in the transform